git-tar/function/handler.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/openfaas/openfaas-cloud/sdk"
 )
 
+const Source = "gh-tar"
```

Comment:
IMO it's better if this is separated into different commits, keeping each one atomic and per function. Will be easier to follow the changes in the git log and easier to revert or fix if some of the functions is broken. Also better for reviewing. WDYT?

Reply:
As this one - [#18](https://github.com/openfaas/openfaas-cloud/pull/18)

Reply:
Yes smaller separate commits would be ideal - per function. What do you think @ericstoekl - would you have time to do that? We also need to figure out the vendoring issue also mentioned by @s8sg on Slack.

Reply:
@alexellis @iyovcheva Thanks for the feedback. I've separated this PR into a separate commit for each function.